e2e/experience/test_experience_crud.py

```
@@ -26,80 +27,268 @@ def test_experience_crud():
+        updated_position = "Lead Test Architect"
+        updated_description = "Updated: Leading test automation initiatives"
 
         try:
-            # Navigate to Work Experience page
+            # ========================================
+            # STEP 1: Navigate to Work Experience page
+            # ========================================
             print("1. Navigating to Work Experience page...")
             page.goto(f"{BASE_URL}/work-experience")
             page.wait_for_load_state("networkidle")
             page.screenshot(path="/tmp/experience_01_page.png")
```

Ruff (0.14.4)

Probable insecure usage of temporary file or directory: "/tmp/experience_01_page.png"

(S108)